src/schema/resolvers/user.js

```diff
@@ -8,52 +8,29 @@ export default {
     currentUser: (parent, args, context, info) => context.user
   },
   Mutation: {
-    createUser: async (parent, args, context, info) => {
-      return createUserDB(args.input, context)
-    },
     signup: async (parent, args, context, info) => {
 
-      let newUser = await context.models.user.findOne({ where: { username: args.data.username } })
-      let error = 'user already exists'
-      if (!newUser) {
-        newUser = await createUserDB(args.data, context)
-        error = null
-      }
+      let user = await context.models.user.findOne({ where: { username: args.data.username } })
+      if (user) return {authError: 'user exist' }
+
+      const newUser = context.models.user.create(args.data);
       const { id, username } = newUser
-      const jwtUser = jwt.sign({ id, username }, process.env.SECRET, { expiresIn: '1d', subject: id }) 
+      const jwtUser = jwt.sign({ id, username }, process.env.SECRET, { expiresIn: '1d', subject: `${id}` }) 
 
-      return { user: newUser, jwt: jwtUser, authError: error }
+      return { user: newUser, jwt: jwtUser, authError: null }
     },
     signin: async (parent, args, context, info) => {
 
       const newUser = await context.models.user.findOne({ where: { username: args.data.username } })
```

Comment:
Let's rename `newUser` to just `user`, since we are looking for an existing one and not creating a new one at all.